backends/bmv2/psa_switch/psaSwitch.cpp

```diff
@@ -895,8 +931,39 @@ Util::IJson* ExternConverter_Digest::convertExternObject(
 
 void ExternConverter_Hash::convertExternInstance(
     UNUSED ConversionContext* ctxt, UNUSED const IR::Declaration* c,
-    UNUSED const IR::ExternBlock* eb, UNUSED const bool& emitExterns)
-{ /* TODO */ }
+    UNUSED const IR::ExternBlock* eb, UNUSED const bool& emitExterns) {
```

Comment:
eb and c are used, so perhaps you can remove the UNUSED annotation where it is wrong.